{"name": "test_loginFailMethod", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <AppiumFramwork.tests.LoginTest.LoginTest testMethod=test_loginFailMethod>\n\n    @pytest.mark.run(order=3)\n    def test_loginFailMethod(self):\n        cl.allureLogs(\"App Opened\")\n        self.gt.clickLoginBotton()\n        self.gt.enterEmail()\n        self.gt.enterPassword2()\n        self.gt.clickOnLoginB()\n>       self.gt.verifyAdminScreen()\n\nLoginTest.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <AppiumFramwork.pages.LoginPage.LoginPageTest object at 0x00000200157F9640>\n\n    def verifyAdminScreen(self):\n        adminScreen = self.isDisplayed(self._pageTitle,\"text\")\n>       assert adminScreen == True\nE       AssertionError\n\n..\\pages\\LoginPage.py:44: AssertionError"}, "steps": [{"name": "App Opened", "status": "passed", "start": 1653640545020, "stop": 1653640545020}, {"name": "Click on Login Button", "status": "passed", "start": 1653640545735, "stop": 1653640545735}, {"name": "Entered email id", "status": "passed", "start": 1653640547306, "stop": 1653640547306}, {"name": "Entered Password", "status": "passed", "start": 1653640548424, "stop": 1653640548424}, {"name": "Clicked on Login Button in Login Screen", "status": "passed", "start": 1653640549007, "stop": 1653640549007}], "attachments": [{"name": "text", "source": "b20117fc-3184-43c5-b64c-ba5156f99df7-attachment.png", "type": "image/png"}, {"name": "text", "source": "541f3d2f-22cf-4b0d-b2fe-df7f6ce534f3-attachment.png", "type": "image/png"}, {"name": "log", "source": "b3c74441-0a10-4790-ab15-da96e1617df5-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "5f66106f-34b2-4451-870e-7e8a593d7928-attachment.txt", "type": "text/plain"}], "start": 1653640545020, "stop": 1653640575825, "uuid": "b4ff8a7b-61fa-4fca-9ef6-3e35ac7416af", "historyId": "aeb6c84858cb897d59574054d46d09e3", "testCaseId": "1512c76b4b2d60f059541c0f73c45353", "fullName": "TestSuite.LoginTest#test_loginFailMethod", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "BTZVBK3"}, {"name": "thread", "value": "1516-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}